app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -2190,22 +2190,26 @@ protected void onPostExecute(Boolean result) {
                 getAlbums().clearSelectedAlbums();
                 mDrawerLayout.setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED);
                 albumsAdapter.notifyDataSetChanged();
+                Snackbar snackbar =
+                    SnackBarHandler.show(
+                        mDrawerLayout,
```

Comment:
You could make it a function and pass the string as a parameter

Reply:
SnackbarHandler is the function created for this purpose.

Reply:
Reuse the statement pls, it's being used many times.

Reply:
But that is how snackbars are created throughout the app. In some cases, the SnackbarHandler functions are used directly, otherwise a snackbar is returned from the class which is stored and then displayed. It has to be this way in order to adjust for the layout and height of the bottom bar.

If there is any other way, could you please suggest?